src/open_inwoner/plans/tests/test_views.py

```diff
@@ -324,14 +325,22 @@ def test_plan_file_cannot_be_downloaded_by_non_contact(self):
     def test_plan_file_add_contact_can_access(self):
         response = self.app.get(self.file_add_url, user=self.contact)
         self.assertEqual(response.status_code, 200)
-        form = response.forms["document-create"]
+        form = response.forms["document-upload"]
         form["file"] = Upload("filename.txt", b"contents")
-        form["name"] = "file"
         response = form.submit(user=self.user)
         self.assertEqual(response.status_code, 302)
 
         self.assertEqual(self.plan.documents.count(), 1)
 
+    def test_htmx_get_plan_file_add_form_renders_upload_form(self):
+        response = self.app.get(
+            self.file_add_url, user=self.user, extra_environ={"HTTP_HX_REQUEST": "true"}
+        )
+
+        self.assertEqual(response.status_code, 200)
+        self.assertIn('id="document-upload"', response.text)
+        self.assertIn("Upload document", response.text)
```

Comment:
The last assert is true by accident. The actual text (as well as title) on the button is "Upload documenten". Better to target a specific element with `PyQuery` and then do `self.assertEqual(py_query_element.text, "Upload documenten")`. This assert will fail for "Upload document" as it should.